pages/reference-smart-account/guards/setModuleGuard.mdx

```diff
@@ -0,0 +1,63 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `setModuleGuard`
+
+Set Module Guard `moduleGuard` for the Safe. Make sure you trust the module guard.
+
+<Callout type='warning'>
+  Set a module guard that checks transactions initiated by the module before
```

Comment:
I think this should be the difinition instead of the current one and I would remove the current one, which is a bit difficult to understand/too technical.